org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/util/CommentRecorderParser.java

```diff
@@ -203,6 +203,14 @@ public int flushCommentsDefinedPriorTo(int position) {
 		return position;
 	}
 
+	/** (Javadoc)
+	 * Flush the JavaDocComment if it is attached with catch clause
+	 */
+	@Override
+	public void flushJavadocCommentsDefinedPriorTo() {
```

Comment:
Why is it necessary to redefine this method rather than using the one already existing?

Reply:
Done